sys/Cargo.toml

```diff
@@ -19,15 +21,19 @@ link-cplusplus = { version = "1.0", optional = true }
 geos-src = { path = "./geos-src", version = "0.2.1", optional = true }
 
 [build-dependencies]
-pkg-config = "0.3"
+pkg-config = "0.3.25"
+semver = "1.0"
 
 [lib]
 name = "geos_sys"
-crate-type = ["dylib", "rlib"]
 
 [features]
 default = []
-v3_6_0 = []
-v3_7_0 = ["v3_6_0"]
-v3_8_0 = ["v3_7_0"]
 static = ["geos-src", "link-cplusplus"]
+
+v3_6_0 = []
```

Comment:
I just thought: if `3.6.0` is the minimum supported version, you don't need this feature anymore.

Reply:
The feature no longer has a meaningful effect here; does it break anyone if we remove it from `geos-sys`?  
(unfamiliar with deprecation policy in Rust projects of removing features)

The feature is still used in the main crate (though could be removed there in a later PR since that involves code changes).